### PR TITLE
[CLB-58] Create sign up page

### PR DESCRIPTION
**Pull request by Mbotunjac**
Created Sign up page and connected it with database, 
Also, added a link to the Login Page to redirect to sign up if user doesn't have an account

### CODE REVIEW

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -72,6 +72,13 @@
         "jobTitle": "Berufsbezeichnung",
         "selectJobTitle": "Wählen Sie eine Berufsbezeichnung",
         "usa": "USA",
+<<<<<<< HEAD
+        "europe": "Europa",
+        "australia": "Australien",
+        "frontEndDeveloper": "front-end developer",
+        "signUpPageText": "Fühlen Sie sich frei, sich anzumelden",
+        "signUp": "Anmelden"
+=======
```
Comment (bojanab987) [inline public/locales/de/translation.json]: You should not have visible Head on  and those lines.
This not resolved. Please keep changes from previous PR and yours.

File: public/locales/de/translation.json [hunk 2]
```diff
@@ -81,6 +88,7 @@
         "status": "Status",
         "confirmDeleteQuestion": "Sind Sie sicher, dass Sie löschen möchten?",
         "saveChanges": "Änderungen Speichern"
+>>>>>>> 1f863d8758975b87c8fd047c6814584aa15edfc8
```
Comment (bojanab987) [inline public/locales/de/translation.json]: Same here as previous comment

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -75,12 +75,17 @@
         "europe": "Europe",
         "australia": "Australia",
         "frontEndDeveloper": "front-end developer",
+<<<<<<< HEAD
+        "signUpPageText": "Feel free to sign up here",
+        "signUp": "Sign up"
+=======
```
Comment (bojanab987) [inline public/locales/en/translation.json]: Here also please correct

File: public/locales/rs/translation.json [hunk 4]
```diff
@@ -71,6 +71,14 @@
...
         "jobTitle": "Pozicija",
         "selectJobTitle": "Izaberi poziciju",
+<<<<<<< HEAD
+        "usa": "SAD",
+        "europe": "Evropa",
+        "australia": "Australija",
+        "frontEndDeveloper": "front-end developer",
+        "signUpPageText": "Ovde možete napraviti svoj nalog",
+        "signUp": "Prijavite se"
+=======
```
Comment (bojanab987) [inline public/locales/rs/translation.json]: And here

**Review by bojanab987 — CHANGES_REQUESTED** [review]

File: public/locales/de/translation.json (hunk 2 above)
Comment (petarjs) [inline public/locales/de/translation.json]: @Mbotunjac it seems the conflicts have been committed here :) pls solve and push :)

File: public/locales/en/translation.json [hunk 5]
```diff
@@ -75,12 +75,17 @@
...
         "img": "Image",
         "job": "Job",
         "save": "Save",
         "status": "Status",
         "confirmDeleteQuestion": "Are you sure you want to delete?",
         "saveChanges": "Save Changes"
+>>>>>>> 1f863d8758975b87c8fd047c6814584aa15edfc8
```
Comment (petarjs) [inline public/locales/en/translation.json]: Conflicts committed here as well :)

File: public/locales/rs/translation.json [hunk 6]
```diff
@@ -81,6 +89,7 @@
         "status": "Status",
         "confirmDeleteQuestion": "Da li ste sigurni da želite da obrišete?",
         "saveChanges": "Sačuvaj Promene"
+>>>>>>> 1f863d8758975b87c8fd047c6814584aa15edfc8
```
Comment (petarjs) [inline public/locales/rs/translation.json]: ...and here :D

**Review by petarjs — CHANGES_REQUESTED** [review]

File: public/locales/de/translation.json (hunk 1 above)
Comment (Mbotunjac) [reply public/locales/de/translation.json]: Thank you, changes have been resolved.

File: public/locales/de/translation.json (hunk 2 above)
Comment (Mbotunjac) [reply public/locales/de/translation.json]: same as previous, resolved :)

File: public/locales/en/translation.json (hunk 3 above)
Comment (Mbotunjac) [reply public/locales/en/translation.json]: resolved

File: public/locales/rs/translation.json (hunk 4 above)
Comment (Mbotunjac) [reply public/locales/rs/translation.json]: resolved

File: public/locales/de/translation.json (hunk 2 above)
Comment (Mbotunjac) [reply public/locales/de/translation.json]: conflicts have been resolved, should be ok now :)

File: public/locales/en/translation.json (hunk 5 above)
Comment (Mbotunjac) [reply public/locales/en/translation.json]: resolved :)

File: public/locales/rs/translation.json (hunk 6 above)
Comment (Mbotunjac) [reply public/locales/rs/translation.json]: resolved :)

**Pull request merged**